src/main/java/hudson/plugins/git/browser/Stash.java

```diff
@@ -86,6 +87,7 @@ public URL getFileLink(Path path) throws IOException {
     }
 
     @Extension
+    @Symbol("stash")
```

Comment:
The `stash` keyword is already used in Pipeline with a very different meaning than this.  If there is a single namespace for symbols, then this attempt to redefine the `stash` symbol seems likely to fail or cause serious harm.

Since stash is the old name for Bitbucket Server, I recommend reverting this change so that stash does not have a symbol assigned.  It is not frequent enough use to justify the risk of symbol collision.
```suggestion
```

Reply:
Would `gitStashRepositoryBrowser` work? Adding the nature of the Symbol (i.e. Browser versus Trait) is something you've suggested elsewhere. Maybe that would be a good way to get this back?

Reply:
Readability, consistency, and brevity might be hard to come by all at once with some of these. I understand the original guidance to avoid needlessly long Symbol names with redundant info in them. On the other hand, if the pursuit of brevity compromises ease of use, I think it's best to use the long names which follow a consistent pattern. The long names would avoid issues, and users can still type out `$class: Stash` if they want to type less stuff.